internal/server/evaluation/evaluation.go

```diff
@@ -34,6 +34,13 @@ func (s *Server) Variant(ctx context.Context, r *rpcevaluation.EvaluationRequest
 		return nil, err
 	}
 
+	if s.analyticsStore != nil {
+		err := s.analyticsStore.IncrementFlagEvaluation(ctx, r.NamespaceKey, r.FlagKey)
+		if err != nil {
+			s.logger.Info("failure to add metric", zap.String("namespace_key", r.NamespaceKey), zap.String("flag_key", r.FlagKey), zap.Error(err))
+		}
+	}
+
```

Comment:
I am going to do this action asynchronously as it is adding some considerable millisecond latency to the overall request.